openedx/features/pakx/lms/pakx_admin_app/constants.py

```diff
@@ -1,6 +1,7 @@
-GROUP_TRAINING_MANAGERS = "Training Manager"
+GROUP_TRAINING_MANAGERS = 'Training Manager'
+GROUP_ORGANIZATION_ADMIN = 'Organization Admins'
 
-ADMIN = 1
+ORG_ADMIN = 1
```

Comment:
what is the diff between `ORG_ADMIN` and `GROUP_ORGANIZATION_ADMIN`

Reply:
GROUP_ORGANIZATION_ADMIN is the group name for organization admins while ORG_ADMIN is an integer value assigned to this role. This value is used b/w backend & frontend in request/response to filter and send user's details.